ios/NebulaNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -7,7 +7,7 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
     private var networkMonitor: NWPathMonitor?
     
     private var site: Site?
-    private var log = Logger(subsystem: "net.defined.mobileNebula", category: "PacketTunnelProvider")
+    static let log = Logger(subsystem: "net.defined.mobileNebula", category: "PacketTunnelProvider")
```

Comment:
also, any reason to change the visibility from private to public?

Reply:
I was having problems with logs not being sent when inside of a subroutine if I wasn't using a static variable, and this is how at least some of the blog posts I was reading set up their logging.  But now that I realize that `.info`, `.debug`, etc don't work from the tunnel provider, maybe that was the issue instead of instance vs static.  I can experiment a bit more with it.

Reply:
I'm not really concerned about the static change so much. It mostly makes sense for this to be a static, I think. But I'd probably opt to make it `private static let log =` if Swift allows you to.

Reply:
I tested out an instance variable, and it worked fine.